jenkins_jobs/modules/wrappers.py

```diff
@@ -1060,17 +1060,28 @@ def build_name(registry, xml_parent, data):
     :arg str name: Name for the build.  Typically you would use a variable
                    from Jenkins in the name.  The syntax would be ${FOO} for
                    the FOO variable.
+    :arg str description: Build description for the build.
+    :arg str run-at-start: Set build name before build starts.
+    :arg str run-at-end: Set build name after build ends.
 
     Example::
 
       wrappers:
         - build-name:
             name: Build-${FOO}
+            description: lorem ipsum dolor
+            run-at-start: true
+            run-at-end: false
     """
     bsetter = XML.SubElement(
         xml_parent, "org.jenkinsci.plugins.buildnamesetter." "BuildNameSetter"
     )
-    mapping = [("name", "template", None)]
+    mapping = [
+        ("name", "template", None),
+        ("description", "descriptionTemplate", ""),
+        ("run-at-start", "runAtStart", None),
+        ("run-at-end", "runAtEnd", None),
```

Comment:
This is change of API... You now make the above parameters mandatory, which will make all existing jobs using this wrapper fail, right?

I can see in Jenkins UI that both options are selected by default so maybe it's good to do the same here too?

Reply:
You are right, I will make them `True` instead of `None`.